src/SFML/Graphics/Buffer.cpp

```diff
@@ -0,0 +1,60 @@
+////////////////////////////////////////////////////////////
+//
+// SFML - Simple and Fast Multimedia Library
+// Copyright (C) 2007-2023 Laurent Gomila (laurent@sfml-dev.org)
+//
+// This software is provided 'as-is', without any express or implied warranty.
+// In no event will the authors be held liable for any damages arising from the use of this software.
+//
+// Permission is granted to anyone to use this software for any purpose,
+// including commercial applications, and to alter it and redistribute it freely,
+// subject to the following restrictions:
+//
+// 1. The origin of this software must not be misrepresented;
+//    you must not claim that you wrote the original software.
+//    If you use this software in a product, an acknowledgment
+//    in the product documentation would be appreciated but is not required.
+//
+// 2. Altered source versions must be plainly marked as such,
+//    and must not be misrepresented as being the original software.
+//
+// 3. This notice may not be removed or altered from any source distribution.
+//
+////////////////////////////////////////////////////////////
+
+////////////////////////////////////////////////////////////
+// Headers
+////////////////////////////////////////////////////////////
+#include <SFML/Graphics/Buffer.h>
+#include <SFML/Graphics/BufferStruct.h>
+#include <SFML/Internal.h>
+
+
+////////////////////////////////////////////////////////////
+CSFML_GRAPHICS_API sfBuffer* sfBuffer_create(void)
+{
+    return new sfBuffer;
+}
+
+
+////////////////////////////////////////////////////////////
+CSFML_GRAPHICS_API void sfBuffer_destroy(sfBuffer* buffer)
+{
+    delete buffer;
+}
+
+
+////////////////////////////////////////////////////////////
+CSFML_GRAPHICS_API size_t sfBuffer_getSize(const sfBuffer* buffer)
+{
+    CSFML_CHECK_RETURN(buffer, 0);
+    return buffer->vector.size();
+}
+
+
+////////////////////////////////////////////////////////////
+CSFML_GRAPHICS_API const sfUint8* sfBuffer_getData(const sfBuffer* buffer)
+{
+    CSFML_CHECK_RETURN(buffer, NULL);
+    return buffer->vector.data();
```

Comment:
`data()` is C++11, I guess we should keep CSFML 2.6 aligned with the C++ versioning if possible

Reply:
Well spotted! Fixed.